web/src/admin/policies/geoip/GeoIPPolicyForm.ts

```diff
@@ -0,0 +1,125 @@
+                        type="checkbox"
+                        ?checked=${this.instance?.executionLogging ?? false}
+                    />
+                    <span class="pf-c-switch__toggle">
+                        <span class="pf-c-switch__toggle-icon">
+                            <i class="fas fa-check" aria-hidden="true"></i>
+                        </span>
+                    </span>
+                    <span class="pf-c-switch__label">${msg("Execution logging")}</span>
+                </label>
+                <p class="pf-c-form__helper-text">
+                    ${msg(
+                        "When this option is enabled, all executions of this policy will be logged. By default, only execution errors are logged.",
+                    )}
+                </p>
+            </ak-form-element-horizontal>
+            <ak-form-group .expanded=${true}>
+                <span slot="header"> ${msg("Policy-specific settings")} </span>
+                <div slot="body" class="pf-c-form">
+                    <ak-form-element-horizontal label=${msg("ASNs")} name="asns">
+                        <input
+                            type="text"
+                            value="${this.instance?.asns ?? ""}"
+                            class="pf-c-form-control"
+                        />
+                        <p class="pf-c-form__helper-text">
+                            ${msg(
+                                "List of autonomous system numbers. Comma separated. E.g. 13335,15169,20940",
+                            )}
+                        </p>
+                    </ak-form-element-horizontal>
+                    <ak-form-element-horizontal label=${msg("Countries")} name="countries">
+                        <ak-dual-select-provider
+                            .provider=${(page: number, search?: string): Promise<DataProvision> => {
+                                return new Iso3166Api(DEFAULT_CONFIG)
+                                    .iso3166List()
+                                    .then((results) => {
+                                        if (!search) return results;
+                                        return results.filter((result) =>
+                                            result.name
+                                                .toLowerCase()
+                                                .includes(search.toLowerCase()),
+                                        );
+                                    })
+                                    .then((results) => {
+                                        return {
+                                            options: results.map((country) => [
+                                                country.code,
+                                                country.name,
+                                            ]),
+                                        };
+                                    });
+                            }}
```

Comment:
This currently hits the API for no reason every time there's a change in `search`. I'll add an optimization.